crates/aptos/src/common/utils.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{CliResult, Error};
+use serde::Serialize;
+
+/// Prompts for confirmation until a yes or no is given explicitly
+/// TODO: Capture interrupts
+pub fn prompt_yes(prompt: &str) -> bool {
```

Comment:
same comment as above ... could imagine one file named results.rs and another confirm.rs but it might be overkill

Reply:
Yeah, which is why I jammed them into categories rather than have a file per type